File: LLM-Translations/GPT-4/codenet/C/Java/s858795471.java
import java.util.Scanner;

public class s858795471 {
    public static boolean search(int[] A, int n, int key) {
        int i = 0;
        A[n] = key;
        while (A[i] != key) i++;
        return i != n;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] A = new int[n + 1];
        for (int i = 0; i < n; i++) {
            A[i] = sc.nextInt();
        }
        int q = sc.nextInt();
        int key, sum = 0;
        for (int i = 0; i < q; i++) {
            key = sc.nextInt();
            if (search(A, n, key)) sum++;
        }
        System.out.println(sum);
    }
}
// 

File: LLM-Translations/GPT-4/codenet/C/Go/s858795471.go
package main

import "fmt"

func search(A []int, n int, key int) int {
	i := 0
	A[n] = key
	for A[i] != key {
		i++
	}
	if i != n {
		return 1
	}
	return 0
}

func main() {
	var n, q, key, sum int

	fmt.Scan(&n)
	A := make([]int, n+1)
	for i := 0; i < n; i++ {
		fmt.Scan(&A[i])
	}
	fmt.Scan(&q)
	for i := 0; i < q; i++ {
		fmt.Scan(&key)
		sum += search(A, n, key)
	}
	fmt.Println(sum)
}

// 
